webmails/rainloop/Dockerfile

```diff
@@ -1,51 +1,65 @@
-ARG ARCH=""
-ARG QEMU=other
+ARG DISTRO=nginx:1.21-alpine
```

Comment:
It makes me nervous when we don't pin a specific alpine version... are we sure that the tag isn't going to "move" ?

Reply:
I agree to pin to specific version. 

The official Docker nginx images do not have frozen version such as ~~nginx:1.21-alpine3.14~~. See https://hub.docker.com/_/nginx?tab=tags&page=1&ordering=last_updated&name=alpine

This is the Dockerfile source:
https://github.com/nginxinc/docker-nginx/blob/master/Dockerfile-alpine.template

My proposal is to use `alpine:3.14` and install required PHP packages + Python3.9 for `socrate`. Then all packages are have a frozen version. I'll investigate this further.